trunk/SUAVE/Components/Energy/Networks/Turboprop_Constant_Speed.py

```diff
@@ -0,0 +1,158 @@
+## @ingroup Components-Energy-Networks
+# Turboprop_Constant_Speed.py
+# 
+# Created:  Jan 2022, S. Karpuk
+# Modified: 
+
+
+# ----------------------------------------------------------------------
+#  Imports
+# ----------------------------------------------------------------------
+
+# package imports
+import numpy as np
+from .Network import Network
+from SUAVE.Components.Physical_Component import Container
+from SUAVE.Core import Data, Units
+
+# ----------------------------------------------------------------------
+#  Network
+# ----------------------------------------------------------------------
+## @ingroup Components-Energy-Networks
+class Turboprop_Constant_Speed(Network):
```

Comment:
I see. Indeed this script is identical to the ICE. The only difference is inside the  turboprop converter (https://github.com/stankarpuktubs/SUAVE/blob/develop/trunk/SUAVE/Components/Energy/Converters/Turboprop.py) where I am using the power lapse formulation for a turboprop. Shall I then modify the ICE network to work with the turboprop converter to avoid unnecessary copies of the code?